scripts/generate_trust_policy

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
Can we use `/bin/bash` to match up with the other shell files?

Reply:
`env bash` is safer for larger distribution as some distros place bash in different places. The other ones should be using `env bash` but it's not as impactful since we control the environment.

https://stackoverflow.com/questions/21612980/why-is-usr-bin-env-bash-superior-to-bin-bash